guides/v3.5.0/tutorial/service.md

```diff
@@ -1,99 +1,52 @@
-To implement this feature, we will take advantage of several Ember concepts:
+To implement this feature, we plan to make use of Ember services.
 
-1. A utility function to create a map from the Google Maps API.
-2. A service to keep a cache of rendered maps to use in different places in the application.
-3. A component to display a map on each rental listing.
+An [Ember Service](https://guides.emberjs.com/release/applications/services/) is an object that can be made available for different parts of an application.
 
-### Making Google Maps Available
+We plan to use the following services to provide maps.
 
-Before implementing a map, we need to make a 3rd party map API available to our Ember app.
-There are several ways to include 3rd party libraries in Ember.
-See the guides section on [managing dependencies](../../addons-and-dependencies/managing-dependencies/)
-as a starting point when you need to add one.
+1. A service that creates a map from our third party map API.
+1. A service that fetches the GPS coordinates of a rental from its location name by consulting a geocoding service.
+1. A service that provides a map to a given element based on a location, and that keeps a cache of rendered maps to use in different places in the application.
 
+The first two services are provided through an Ember addon we plan to install called [ember-simple-leaflet-maps](https://github.com/ember-learn/ember-simple-leaflet-maps).
 
-The [Google Maps API](https://developers.google.com/maps/documentation/javascript/tutorial/) requires us to reference its library from a remote script.
-In this case we'll provide this script to our Ember app via an addon called `ember-simple-google-maps`.
+We will use these two services in the third service that we will create here as part of the tutorial.
 
-```bash
-ember install ember-simple-google-maps
-```
+### Making Maps Available
 
-Google Maps requires an API key for deployment.
-You can [Generate an API key](https://developers.google.com/maps/documentation/javascript/get-api-key/)
-from Google.
-Add your new API key to the application by stopping the server and restarting it with the environment variable, `GOOGLE_MAPS_API_KEY`.
+Before implementing a map, we need to make a 3rd party map API available to our Ember app.
+There are several ways to include 3rd party libraries in Ember.
+See the guides section on [managing dependencies](../../addons-and-dependencies/managing-dependencies/) as a starting point when you need to add one.
+
+For the purpose of this tutorial we will take advantage of the [Leaflet Maps API](https://leafletjs.com/)
+The [Leaflet Maps API](https://leafletjs.com/) requires us to reference its library from a remote script.
+In this case we'll provide this script to our Ember app via an addon called `ember-simple-leaflet-maps`.
 
 ```bash
-GOOGLE_MAPS_API_KEY=<your key here> ember s
+ember install ember-simple-leaflet-maps
 ```
 
-### Accessing the Google Maps API with a Utility
```

Comment:
Removing the Utility part of this guides section.  I thought it wasn't necessary because:

* It was too many things introduced at the same time
* It was dealing directly with the maps API
* Removing it reduces the size/complexity of the section.

Reply:
This is a good move! Totally agree